app/user-setup/index.tsx

```diff
@@ -72,28 +66,35 @@ function UserSetupPage() {
 
             if (!res.canceled) {
                 setValue("imageUrl", res.assets[0].uri);
-                setImageURI(res.assets[0].uri);
             }
         } catch (error) {
             console.error(error);
         }
     }
 
+    function handleRemovePhoto() {
+        const defaultImageURI = user?.photoURL ?? "";
+        setValue("imageUrl", defaultImageURI);
+    }
+
     function handleSetCampus(campus: string) {
         setValue(
             "campus",
             CampusEnum.options[0] === campus
                 ? CampusEnum.options[0]
                 : CampusEnum.options[1],
         );
-        setCampusSelected(campus);
     }
 
     async function handleSetupUser(data: SetUpUserRequestType) {
         setIsLoading(true);
         try {
-            await uploadImages([imageURI], AVATAR_IMAGES_FOLDER);
-            await setUpUser(data);
+            await uploadImages([imageURI ?? ""], AVATAR_IMAGES_FOLDER).then(
+                async (value) => {
+                    data.imageUrl = value[0];
+                    await setUpUser(data);
+                },
+            );
```

Comment:
why do we do it this way? we have an `await` and `.then`. we dont want to use `.then` and callbacks in general, lets use `await`.

it doesnt work this way?
```tsx
const images = await uploadImages([imageURI], AVATAR_IMAGES_FOLDER);
data.imageUrl = images[0];
await setUpUser(data);
```

Reply:
naa to syay error last time i tried it, ako tryan balik

Reply:
ahh okay ang problem before kay ang na index is ang uploadImages, muwork na